pxtcompiler/emitter/decompiler.ts

```diff
@@ -302,7 +302,7 @@ namespace ts.pxtc.decompiler {
 
     export function decompileToBlocks(blocksInfo: pxtc.BlocksInfo, file: ts.SourceFile, options: DecompileBlocksOptions, renameMap?: RenameMap): pxtc.CompileResult {
         let emittedBlocks = 0;
-        let stmts: ts.Statement[] = file.statements;
+        let stmts: NodeArray<ts.Statement> = file.statements;
```

Comment:
is this a new type?

Reply:
No, but the type definition in the TS compiler changed so it can no longer be assigned to a normal array. Pretty sure the reason is that the indices are `readonly` in the `NodeArray` (which is a TS 2.0 feature).